_partners/gsi-technology.md

```diff
@@ -0,0 +1,6 @@
+---
+name: GSI Technology
+# upload your logo to the following directory - must be square
+logo: '/assets/media/partners/just-gsi-logo.png'
+link: 'https://gsitechnology.com/apu'
```

Comment:
Humm. That URL doesn't seem to work.

Reply:
Double checked - https://gsitechnology.com/apu works fine - here is a screenshot 
<img width="1422" alt="Screen Shot 2021-07-07 at 11 22 28" src="https://user-images.githubusercontent.com/47303277/124725652-b04bd880-df15-11eb-8881-71b0730e4951.png">

Reply:
https://gsitechnology.com/apu seems to be working fine  - here is a screenshot:
<img width="1422" alt="Screen Shot 2021-07-07 at 11 22 28" src="https://user-images.githubusercontent.com/47303277/124726044-16d0f680-df16-11eb-87db-f00032034dd6.png">

Reply:
www added